docs/content/Examples.md

```diff
@@ -6,86 +6,77 @@ redirect_from:
-| [Event Analytics][link-event-analytics]         |     [event-analytics][code-event-analytics]     | Mixpanel like Event Analytics App built with Cube.js and Snowplow                         |
-| [External Rollups][link-external-rollups]       |    [external-rollups][code-external-rollups]    | Compare performance of direct BigQuery querying vs MySQL cached version for the same data |
-| Simple Dynamic Schema Creation                  | [async-module-simple][code-simple-asyncmodule]  | A simple example of using `asyncModule` to generate schemas                               |
-| Auth0                                           |               [auth0][code-auth0]               | Cube.js deployment configured with Auth0 JWK/JWT integration                              |
-| Cognito                                         |             [cognito][code-cognito]             | Cube.js deployment configured with AWS Cognito JWK/JWT integration                        |
-
-[link-real-time-dashboard]: https://real-time-dashboard-demo.cube.dev/
-[code-real-time-dashboard]:
-  https://github.com/cube-js/cube.js/tree/master/examples/real-time-dashboard
-[link-react-dashboard]: https://react-dashboard-demo.cube.dev/
-[code-react-dashboard]:
-  https://github.com/cube-js/cube.js/tree/master/guides/react-dashboard/demo
-[link-d3-dashboard]: https://d3-dashboard-demo.cube.dev/
-[code-d3-dashboard]:
-  https://github.com/cube-js/cube.js/tree/master/examples/d3-dashboard
-[link-stripe-dashboard]:
-  http://cubejs-stripe-dashboard-example.s3-website-us-west-2.amazonaws.com/
-[code-stripe-dashboard]:
-  https://github.com/cube-js/cube.js/tree/master/examples/stripe-dashboard
-[link-event-analytics]: https://d1ygcqhosay4lt.cloudfront.net/
-[code-event-analytics]:
-  https://github.com/cube-js/cube.js/tree/master/examples/event-analytics
-[link-external-rollups]: https://external-rollups-demo.cube.dev/
-[code-external-rollups]:
-  https://github.com/cube-js/cube.js/tree/master/examples/external-rollups
-[link-web-analytics]: https://web-analytics-demo.cube.dev/
-[code-web-analytics]:
-  https://github.com/cube-js/cube.js/tree/master/examples/web-analytics
-[code-simple-asyncmodule]:
-  https://github.com/cube-js/cube.js/tree/master/examples/async-module-simple
-[code-auth0]: https://github.com/cube-js/cube.js/tree/master/examples/auth0
-[code-cognito]: https://github.com/cube-js/cube.js/tree/master/examples/cognito
+Below you can find tutorials to help you get started with Cube.js.
+<!-- Also, please review the [recipes](#recipes) for common use cases. -->
+
 
 ## Tutorials
 
-### Getting Started Tutorials
+These tutorials are great to start learning Cube.js:
+
+* [Cube.js, the Open Source Dashboard Framework: Ultimate Guide](https://cube.dev/blog/cubejs-open-source-dashboard-framework-ultimate-guide) —
+  you will launch an API with Cube.js and build a dashboard with React
```

Comment:
@igorlukanin WDYT about removing all the 'you will' bits from the following bullet points?

```suggestion
  launch an API with Cube.js and build a dashboard with React
```

Reply:
Great suggestion, fixed!